client/apps/admin-portal/src/app/manage/manage.component.html

```diff
@@ -1 +1 @@
-<p>manage works!</p>
+<sb-searchbar></sb-searchbar>
```

Comment:
The search bar doesn't belong in the manage component. See this link for what the manage component is for https://xd.adobe.com/view/b3ecccb5-ea3f-4b80-5d18-581c6b04f72d-c5dd/screen/9610b7bd-a4fe-41c3-a89f-32d56a2835b5/Manage/?fullscreen

The search bar should live in the sermon component.